src/components/RenderRouter/ArchiveItem.tsx

```diff
@@ -0,0 +1,269 @@
+
+import React, { EventHandler, SyntheticEvent } from 'react';
+import { withRouter, RouteComponentProps } from 'react-router-dom';
+import "./ArchiveItem.scss"
+import Amplify from 'aws-amplify';
+import awsmobile from '../../aws-exports';
+import DataLoader, {
+    VideoByVideoTypeData,
+    SeriesByTypeData
+} from './DataLoader';
+import VideoOverlay from '../VideoOverlay/VideoOverlay';
+import format from 'date-fns/format';
+
+Amplify.configure(awsmobile);
+
+type listData = SeriesByTypeData | VideoByVideoTypeData
+
+interface Props extends RouteComponentProps {
+    content: any,
+    data: any,
+    pageConfig: any,
+}
+interface State {
+    listData: listData[]
+    overlayData: any
+    urlHistoryState: any
+    selectedYear: string
+    yearOptions: string[]
+    currentNextToken: string | null
+}
+class ArchiveItem extends React.Component<Props, State> {
+
+    constructor(props: Props) {
+        super(props);
+        this.state = {
+            listData: [],
+            overlayData: null,
+            urlHistoryState: window.history.state,
```

Comment:
You're using urlHistoryState to store the history URL, but you initialize it with the history's state. This should probably just be `string | null` and I assume if it's `null` when you close the overlay you just do nothing.